README.md

```diff
@@ -34,6 +34,32 @@ Run the application by executing:
 This repo is now integrated with the rse-cft-library. For more information see the bootWithCCD task in build.gradle.
 Extra compose files can be provided in src/cftlib/resources/docker.
 
+#### Cftlib - Docker Desktop v4.30
+
+**_First Time Setup_**
+
+As part of this setup the definitions repository has been added as a git submodule.
+1. `cd definitions`
+2. `git submodule init` & `git submodule update`
+4. `git checkout master` & `git pull`
+
+You should also run `docker pull hmctspublic.azurecr.io/hmcts/rse/rse-idam-simulator:latest`
+
+_Ensure you have removed all existing / leftover containers_
+
+**On Mac**
+
+1. `sudo nano /etc/hosts`

```

Comment:
Same with these, worth moving them to a simple script and make it idempotent. Something like

```
HOST_ENTRY="127.0.0.1    rse-idam-simulator"

if grep -q "$HOST_ENTRY" /etc/hosts; then
  echo "Entry already exists in /etc/hosts"
else
  sudo bash -c "echo '$HOST_ENTRY' >> /etc/hosts"
fi
```

Reply:
Added in start.sh